examples/wordle/src/App.tsx

```diff
@@ -174,6 +174,16 @@ function App() {
           <li>An incorrect letter stays black.</li>
         </ul>
       </div>
+      <div className="cs11 ce12">
+        <div className="feedback">
+          <a
+            target="_top"
+            href="https://github.com/miroapp/app-examples/issues"
+            title="Leave feedback"
```

Comment:
Why doesnt this link have content?

Reply:
@marcospinello Worked on this one - I think... @marcospinello  please confirm...